src/common/SurgeSynthesizer.cpp

```diff
@@ -19,8 +20,11 @@
 #elif TARGET_APP
 #include "PluginLayer.h"
 #include "vstgui/plugin-bindings/plugguieditor.h"
+#elif TARGET_HEADLESS
+#include "HeadlessPluginLayerProxy.h" // from src/headless
```

Comment:
I think the comment serves no purpose. Like why we do it for this file and not for all files. Or all target specific files.

Reply:
Yup